console-subscriber/src/lib.rs

```diff
@@ -263,6 +263,7 @@ impl ConsoleLayer {
         // The `cfg` value *appears* to be a constant to clippy, but it changes
         // depending on the build-time configuration...
         #![allow(clippy::assertions_on_constants)]
+        #[cfg(not(console_without_tokio_unstable))]
         assert!(
             cfg!(tokio_unstable),
```

Comment:
We can combine these 2 together so that it asserts only if neither are set.

```suggestion
        assert!(
            cfg!(any(tokio_unstable, console_without_tokio_unstable)),
```

Reply:
nice, didn't expect cfg! can do this.